app/models/publication.rb

```diff
@@ -80,6 +80,7 @@ def self.open_access_statuses
         }
 
   scope :subject_to_open_access_policy, -> { journal_article.published.where('published_on >= ?', Publication::OPEN_ACCESS_POLICY_START) }
+  scope :claimable_by, ->(user) { journal_article.visible.where.not(id: user.authorships.unclaimable.map { |a| a.publication.id }) }
 
```

Comment:
Just realized that I ended up with a double negative here. Works fine. Doesn't read that well though..